packages/mdc-snackbar/README.md

```diff
@@ -59,47 +63,76 @@ npm install @material/snackbar
 
 ### JavaScript Instantiation
 
-MDC Snackbar ships with a Component / Foundation combo which provides the API for showing snackbar messages with optional action.
-
 ```js
 import {MDCSnackbar} from '@material/snackbar';
-
 const snackbar = new MDCSnackbar(document.querySelector('.mdc-snackbar'));
 ```
 
 > See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
 
 ## Variants
 
-### Start Aligned Snackbars (tablet and desktop only)
+### Leading (tablet and desktop only)
```

Comment:
Not related to this PR but, we should also provide a mixin to set position that can be used with media queries.

Reply:
Not sure exactly what you mean - could you give me an example?

Is your goal to allow clients to automatically switch to `--leading` on large screens?

Reply:
Here is an example in case if user wants to position the snackbar for bigger screens using media queries:

```
<div class="mdc-snackbar snackbar-example">Example toast message</div>

@media (min-width:1200px) {
  .snackbar-example {
    @include mdc-snackbar-position-leading;
  }
}
```

Helper class (`--leading`) is good but I think snackbar will not be able to dynamically change position based on device width. WDYT?

Reply:
Done